test/GnosisSafeSameAddressMultisig.js

```diff
@@ -39,6 +46,19 @@ describe("GnosisSafeSameAddressMultisig", function () {
     });
 
     context("Verifying multisigs", async function () {
+        it("Try to deploy a contract without specified proxies or zero hashes", async function () {
+            const GnosisSafeSameAddressMultisig = await ethers.getContractFactory("GnosisSafeSameAddressMultisig");
+
+            await expect(
+                GnosisSafeSameAddressMultisig.deploy([])
+            ).to.be.revertedWithCustomError(gnosisSafeSameAddressMultisig, "ZeroValue");
+
+            const bytes32Zero = "0x" + "0".repeat(64);
+            await expect(
+                GnosisSafeSameAddressMultisig.deploy([bytes32Zero])
+            ).to.be.revertedWithCustomError(gnosisSafeSameAddressMultisig, "ZeroValue");
+        });
+
```

Comment:
we should also update the tests regarding the GnosisSafeSameAddressMultisig.create() method, no?

Reply:
There is one in the `serviceStaking` testing